recipes/lora_finetune_distributed.py

```diff
@@ -181,10 +181,14 @@ def __init__(self, cfg: DictConfig) -> None:
                 raise RuntimeError(
                     "enable_activation_offloading should only be True when enable_activation_checkpointing is True"
                 )
-        elif self._enable_activation_checkpointing:
-            log.info(
+        elif (
+            self._enable_activation_checkpointing
+            and cfg.checkpointer.model_type != "LLAMA3_VISION"
```

Comment:
what happens if someone tries to use activation offloading with a vision model? broken?

Reply:
it will already error out elsewhere with a descriptive message. but it's odd to see "try activation offloading" and then "activation offloading is not supported for multimodal"